extensions/amp-consent/0.1/consent-config.js

```diff
@@ -244,6 +246,26 @@ export class ConsentConfig {
     return mergedConfig;
   }
 
+  /**
+   * Validate if story consent then no promptUiSrc
+   * @param {!JsonObject} config
+   * @return {!JsonObject}
+   */
+  checkStoryConsent_(config) {
+    if (childElementByTag(
+      this.element_,
+      AMP_STORY_CONSENT_TAG
+    )) {
+      userAssert(
+        !config["promptUISrc"],
```

Comment:
nit: I believe it's exactly the same, but please use single quotes for style consistency : )